apps/src/music/views/timeline.module.scss

```diff
@@ -71,12 +71,20 @@
     height: 100%;
 
     .timelineElement {
+      margin: 0;
+      padding: 0;
       position: absolute;
       box-sizing: border-box;
       border-radius: 8px;
       border-width: 2px;
       border-style: solid;
 
+      &:focus-visible {
+        outline: 1px solid $light_white;
+        outline-offset: 2px;
+        border-radius: 0.375rem;
```

Comment:
Just curious: is there a reason to do this in `rem` when other items are in `px`?

Reply:
This seems to be the pattern in other focus-visible places in our codebase. Generally what I learned from @kelbyhawn and @moshebaricdo is that we should default to rem unless it's something tiny and specific (like an outline)